include/overlay021/ov21_021D57B4.h

```diff
@@ -7,13 +7,13 @@
 
 #include "cell_actor.h"
 
-void ov21_021D57B4(UnkStruct_ov21_021D5844 *param0, int param1, int param2, int param3);
-void ov21_021D5844(UnkStruct_ov21_021D5844 *param0);
+void ov21_021D57B4(pokedexEncStruct *dexEncData, int param1, int param2, int param3);
+void Free_pokedexEncData(pokedexEncStruct *dexEncData);
```

Comment:
**nomenclature**: Always start with the module name, then the action when labeling functions in this manner.